### PR TITLE
[ Bug ] Fix the bug read the weight for batch normalization layer

### PR DESCRIPTION
**Pull request by jijoongmoon**
There is bug when the model loads the data for the batch normalziation layer.

During the setup the requestWeights in manager, it add the max execution order for graddient for gradient clipping, but variable weight also added. This pr fixs it.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: nntrainer/tensor/manager.cpp [hunk 1]
```diff
@@ -398,7 +398,7 @@ std::vector<Weight *> Manager::requestWeights(
      */
     if (Weight::isGradientClipByGlobalNorm(clip_by_global_norm)) {
       grad_exec_order.push_back(TensorPool::PERSIST_END_ORDER);
-      var_exec_order.push_back(TensorPool::PERSIST_END_ORDER);
+      // var_exec_order.push_back(TensorPool::PERSIST_END_ORDER);
```
Comment (myungjoo) [inline nntrainer/tensor/manager.cpp:401]: Is there any reason not to delete the line, but to comment it out?

If this is really unnecessary, leaving it commented will confuse other developers a few months later.

Comment (jijoongmoon) [reply nntrainer/tensor/manager.cpp:401]: I add a comment. Thanks.

**Review by myungjoo — APPROVED** [review]

**Review by baek2sm — APPROVED** [review]
LGTM

**Comment by SeoHyungjun** [comment]
LGTM
